src/reference/interactive/index.pug

```diff
@@ -21,7 +21,7 @@ block reference
 
     h2#introduction Introduction
 
-    p Beam Interactive is a new way to let viewers directly control the environment in and around streamer’s broadcasts by letting viewers interact using a user interface. When a broadcast has interactivity enabled visual controls will appear beneath the broadcast's video that respond directly to the events of the broadcast. These controls can dynamically update as the broadcast progresses in response to different situations that occur.
+    p Beam Interactive is a new way to let viewers directly control the environment in and around streamer’s broadcasts by letting viewers interact using a user interface. When a broadcast has interactivity enabled visual controls will appear beneath the broadcast's video that responds directly to the events of the broadcast. These controls can dynamically update as the broadcast progresses in response to different situations that occur.
```

Comment:
The previous version was correct. `The people respond to` vs `The people responds to`.